site/src/docs/components/header/index.mdx

```diff
@@ -436,109 +436,190 @@ The Header.NavigationMenu is enables users to navigate to different pages of the
+          <Header.Search label="Hae palvelusta" onChange={() => {}} onSubmit={() => {}} />
+        </Header.ActionBarItem>
+      </Header.ActionBar>
+      <Header.NavigationMenu>
+        <Header.Link
+          href="#"
+          label="Sosiaali- ja terveyspalvelut"
+          dropdownLinks={[
+            <Header.Link
+              as={Link}
+              href="#"
+              label="Terveydenhoito"
+              active
+              dropdownLinks={[
+                <Header.Link as={Link} href="#" label="Hammashoito" />,
+                <Header.Link as={Link} href="#" label="Julkinen terveydenhoito" />,
+              ]}
+            />,
+            <Header.Link
+              as={Link}
+              href="#"
+              label="Senioripalvelut"
+              dropdownLinks={[
+                <Header.Link as={Link} href="#" label="Viriketoiminta" />,
+                <Header.Link as={Link} href="#" label="Kuntouttavat palvelut" />,
+              ]}
+            />,
+          ]}
+        />
+        <Header.Link href="#" label="Kasvatus ja koulutus" />
+        <Header.Link href="#" label="Asuminen" />
+      </Header.NavigationMenu>
+    </Header>
+  </div>
+  <div style={{
+    position: "relative",
+    "z-index": "11"
+  }}>
+    <Header theme={{
+      '--color-focus-outline': 'var(--color-metro-dark)',
+      '--header-color': 'var(--color-black)',
+      '--actionbar-background-color': 'var(--color-engel)',
+      '--nav-background-color': 'var(--color-engel-light)',
+      '--nav-border-color': 'var(--color-black)',
+      '--nav-link-hover-color': 'var(--color-black)',
+      '--universal-bar-background-color': 'var(--color-black-20)',
+      '--nav-link-dropdown-background-color': 'var(--color-white)',
+      '--nav-button-background-color': 'var(--color-black-20)',
+      '--nav-button-hover-background-color': 'var(--color-black-40)',
+      '--nav-drop-down-icon-color': 'var(--color-black)',
+      '--header-background-color': 'transparent',
+    }}>
+      <Header.ActionBar
+        title="Helsingin kaupunki"
+        titleHref="https://hel.fi"
+        logoAriaLabel="Service logo"
+        logoHref="https://hel.fi"
+        logo={<Logo src={logoFi} alt="Helsingin kaupunki" />}
+        menuButtonAriaLabel="Menu"
+      >
+        <Header.LanguageSelector
+          languages={[
+            { label: 'Suomi', value: 'fi' },
+            { label: 'Svenska', value: 'sv' },
+            { label: 'English', value: 'en' },
+          ]}
+        />
+        <Header.ActionBarItem label="Haku" icon={<IconSearch />} id="action-bar-search">
+          <Header.Search label="Hae palvelusta" onChange={() => {}} onSubmit={() => {}} />
+        </Header.ActionBarItem>
+      </Header.ActionBar>
+      <Header.NavigationMenu>
+        <Header.Link
+          href="#"
+          label="Sosiaali- ja terveyspalvelut"
+          dropdownLinks={[
```

Comment:
Any of the dropdowns don't have a themed background color.

Reply:
fixed